lib/message.js

```diff
@@ -39,9 +39,8 @@ Message.prototype.magicHash = function magicHash() {
 };
 
 Message.prototype._sign = function _sign(privateKey) {
-  if (!(privateKey instanceof PrivateKey)) {
-    throw new TypeError('First argument should be an instance of PrivateKey');
-  }
+  $.checkArgument(privateKey instanceof PrivateKey,
+    new TypeError('First argument should be an instance of PrivateKey'));
```

Comment:
`checkArgument` creates a new error, so this would create two errors where there could be one. No?